src/sql/parts/notebook/notebookActions.ts

```diff
@@ -38,6 +40,42 @@ export class AddCellAction extends Action {
 	}
 }
 
+export class TrustedAction extends Action {
+	private static readonly trustLabel = localize('trustLabel', 'Trusted');
+	private static readonly notTrustLabel = localize('untrustLabel', 'Not Trusted');
+	private static readonly alreadyTrustedMsg = localize('alreadyTrustedMsg', 'Notebook is already trusted.');
+
+	constructor(
+		id: string, isTrusted: boolean
```

Comment:
Nitpick: why not have all this on 1 line? It'll easily fix.

Reply:
Added notification service injection.